### PR TITLE
Add Logotype and edit Readme

### PR DESCRIPTION
**Pull request by richardbmx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -225,6 +227,7 @@ Special thanks to the authors of:
 - [TypeORM](https://github.com/typeorm/typeorm)
 - [TypeGraphQL](https://github.com/19majkel94/type-graphql)
 - [Prisma](https://github.com/prisma/prisma)
+- [Logo design Richardbmx](https://github.com/richardbmx)
 
 Warthog is essentially a really opinionated composition of TypeORM and TypeGraphQL that uses similar GraphQL conventions to the Prisma project.
```
Comment (goldcaddy77) [inline README.md]: Below: can you put this in an `images` folder and call it `logo.png`

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+![img](https://github.com/richardbmx/warthog/blob/master/github.png?raw=true)
```
Comment (goldcaddy77) [inline README.md]: Once you make change below, you should be able to refer to this relatively `./images/logo.png?raw=true`  Then you don't need to refer to your fork.

**Comment by richardbmx** [comment]
Hello, I have not seen your comment here, github has not been used very well since I am more graphic designer than programmer in this way I was taught and realized.

**Comment by goldcaddy77** [comment]
No worries. I can grab the photo and do the PR! That's for the logo it's amazing!

**Pull request closed without merging**